src/input/mod.rs

```diff
@@ -845,40 +859,50 @@ impl State {
                                     // Don't check override redirect windows, because we don't set keyboard focus to them explicitly.
                                     // These cases are handled by the XwaylandKeyboardGrab.
                                     if let Some(target) = shell.element_under(pos, &output) {
-                                        if seat.get_keyboard().unwrap().modifier_state().logo {
+                                        if seat.get_keyboard().unwrap().modifier_state().alt {
```

Comment:
Why change the modifier?

Reply:
I changed it for testing because the logo key was being caught by my DE, didn't mean to commit it.